parsec/core/mountpoint/fuse_runner.py

```diff
@@ -19,6 +21,21 @@
 logger = get_logger()
 
 
+@contextmanager
+def _reset_signals(signals=None):
+    if signals is None:
+        signals = (signal.SIGINT, signal.SIGTERM, signal.SIGHUP, signal.SIGPIPE)
+    saved = {sig: signal.getsignal(sig) for sig in signals}
+    try:
+        yield
+    finally:
+        for sig, handler in saved.items():
+            try:
+                signal.signal(sig, handler)
+            except ValueError:
+                pass
```

Comment:
Lors de la connexion de mon utilisateur depuis la GUI, l'exception ValueError est déclenchée.
Elle contient "signal only works in main thread". Du coup si on n'appelle pas ce code depuis le thread principal ça ne fonctionne pas...